source/01_title_page.md

```diff
@@ -9,7 +9,6 @@ and add the centred logo
 
     % Delete the following line
     % to remove the UCL header logo
-    \ThisULCornerWallPaper{1.0}{style/univ_logo.eps}
         
```

Comment:
Is it not helpful to have an example to show how a header can be added to this page?